src/cronGenerator.ts

```diff
@@ -0,0 +1,33 @@
+import cronstrue from 'cronstrue';
+
+export function generateCronExpression() {
+    function getRandomOrWildcard(min: number, max: number) {
+        if (Math.random() < 0.7) {
+            return '*';
+        } else {
+            return getRandomIntInclusive(min, max);
+        }
+    }
+    const minute = getRandomOrWildcard(0, 59);
+    const hour = getRandomOrWildcard(0, 23);
+    const dayOfMonth = getRandomOrWildcard(1, 31);
+    const month = getRandomOrWildcard(1, 12);
+    const dayOfWeek = getRandomOrWildcard(0, 6);
+
+    return `${minute} ${hour} ${dayOfMonth} ${month} ${dayOfWeek}`;
+}
+
+export function generateReadableExpression(cronExpression: string): string {
+    try {
+        return cronstrue.toString(cronExpression);
+    } catch (error) {
```

Comment:
Don't try catch here, let it throw error. It should be automatically handled by the global express error catcher and accordingly return an error.